src/platform/site-wide/user-nav/containers/Main.jsx

```diff
@@ -70,8 +73,16 @@ export class Main extends React.Component {
   }
 
   checkLoggedInStatus = () => {
+    ssoKeepAliveSession();
```

Comment:
should we run the `ssoKeepAliveSession` if ssoe is disabled?

Reply:
ah no, good point. I'll add that check

Reply:
the other thing I realized this is it bypasses our self-enforced rate limiting, but for auto-login/logout to work, I don't think there's a way around it.

Example:

I'm logged in on VA.gov. I go to MHV and log out. I go back to VA.gov before the threshold to call keep-alive again passes. That can't be a blocker from calling the endpoint again, since the expected behavior is to trigger a logout.

Granted, that seems like an edge case, but one worth accounting for. I just don't know how that will affect that endpoint. Since user-nav is on every page, it renders the whole rate limiting moot. Not sure if there's an alternative to that approach.

Reply:
@hinzed1127 I think based on Patricks latest conversation with the IAM team we'd only want to make the `ssoKeepAliveSession` call if `canCallSSO` is true